qr-code-auth/README.md

```diff
@@ -1,59 +1,310 @@
+      }
+      ```
+
+- **Example for user authentication document creation request:**
+  POST /qr-code-auth<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
+
+  ```json
+  {
+    "user_id": "SooJK37gzjIZfFNH0tlL",
+    "device_info": "t5k77PHnuDSrgEzvMJAj",
+    "is_authorized": true,
+    "access_token": "NLFSj7Kz30oHgolfIZtJ"
+  }
+  ```
+
+  Response :
+  Status 201<br/>
+  Content-Type: application/json<br/>
+
+  ```json
+  {
+    "data": {
+      "user_id": "SooJK37gzjIZfFNH0tlL",
+      "device_info": "t5k77PHnuDSrgEzvMJAj",
+      "is_authorized": true,
+      "access_token": "NLFSj7Kz30oHgolfIZtJ"
+    },
+    "message": "authentication document created successfully."
+  }
+  ```
+
+## GET /qr-code-auth
+
+Retrieves THE authentication document.
+
+- **Params**
+  None
+- **Query**
+
+  - user_id : Specifies the ID of the User whose authentication document will be retrieved
 
 - **Body**  
- ```
- {
-  "user_id": "<user-id>",
-  "device_info": "<device-info>"
- }
-```
+  None
 - **Headers**  
   None
 - **Cookie**  
   None
+- **Success Response:**
 
-#### Success Response:
+  - **Code:** 200
 
-```
-{
-    "statusCode": 201,
-    "userDeviceInfoData: {
-        "user_id": "9",
-        "device_info": "window"
-    },
-    "message": "User Device Info added successfully!"
-}
-```
-#### Error Responses
-```
-{
-    "statusCode": 404,
-    "message": "Api not found."
-}
-```
-
-```
-{
-    "statusCode": 500,
-    "message": "Internal server error."
-}
-```
-```
-{
+    - **Content:**
+
+    ```json
+    {
+      "message": "String",
+      "count": "Number",
+      "data": [
+        {
+          "user_id": "String",
+          "device_info": "String",
+          "is_authorized": "Boolean",
+          "access_token": "String"
+        }
+      ]
+    }
+    ```
+
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 404
+    - **Content:**
+      ```json
+      {
+        "message": "User with id <:id> does not exist."
+      }
+      ```
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example:**
+  GET /qr-code-auth?user_id=SooJK37gzjIZfFNH0tlL<br/>
+  Status: 200 OK<br/>
+  ```json
+  {
+    "message": "Authentication document retrieved successfully.",
+    "count": 2,
+    "data": [
+      {
+        "user_id": "SooJK37gzjIZfFNH0tlL",
+        "device_info": "t5k77PHnuDSrgEzvMJAj",
+        "is_authorized": true,
+        "access_token": "NLFSj7Kz30oHgolfIZtJ"
+      },
+      {
+        "user_id": "SooJK37gzjIZfFNH0tlL",
+        "device_info": "en7nlNnpfqoqodcmtMeZ",
+        "is_authorized": false,
+        "access_token": "GTB4UUtlKwGemRN2lwBp11"
+      }
+    ]
```

Comment:
Why multiple documents for a single user?

Reply:
As per my discussion with @shreya-mishra . We discussed that a single user can log on from different devices hence multiple documents. @shreya-mishra please could you please confirm if thats true.

Reply:
I rediscussed with @shreya-mishra we will be having a single document. Updated the same. Please check now @prakashchoudhary07